server/controllers/users/getUsersByLang.js

```diff
@@ -1,3 +1,25 @@
-exports.getUsersByLang = (req, res) => {
-  res.send('hi form native lang');
+const { users: { getUsersByLang }, languages: { getLanguageById } } = require('../../database/queries');
+const { formatLanguages } = require('../../helpers/formatLanguages');
+
+exports.getUsersByLang = (req, res, next) => {
+  let users;
+  const { id } = req.params;
+
+  if (!Number(id)) return next({ code: 400, msg: 'Bad request' });
+
+  getUsersByLang(id)
+    .then(({ rows }) => {
+      users = rows;
+      return users.map((user) => getLanguageById(user.native_lang_id));
```

Comment:
```suggestion
return Promise.all(
    users.map((user) => getLanguageById(user.native_lang_id))
);
```
then you can get rid of the below line